src/handlers.rs

```diff
@@ -423,6 +430,44 @@ fn handle_read_file_sync(
   ))
 }
 
+fn handle_metadata_sync(
+  _d: *const DenoC,
+  builder: &mut FlatBufferBuilder,
+  filename: &str,
+) -> HandlerResult {
+  debug!("handle_metadata_sync {}", filename);
+  let metadata = fs::symlink_metadata(Path::new(filename))?;
```

Comment:
If you use this, I guess the javascript method should be called lstatSync.

Reply:
@piscisaureus I don't think functionally it will make difference if we're not just returning what `lstat` and `stat` return?

Reply:
Hmm, actually on thinking about this I agree it should be called `lstatSync` or `symlinkStatSync` or something along those lines.

Reply:
I can add another binding that always follows or should that be a separate PR?

Reply:
@Aaronepower Probably best you implement both `statSync` and `lstatSync` now ... ideally it won't require twice the amount of code....
@piscisaureus good catch